fxtest/fxtest.go

```diff
@@ -34,6 +36,50 @@ type TB interface {
 	FailNow()
 }
 
+type testPrinter struct {
+	TB
+}
+
+func (p *testPrinter) Printf(format string, args ...interface{}) {
+	p.Logf(format, args...)
+}
+
+// App is a wrapper around fx.App that provides some testing helpers. By
+// default, it uses the provided TB as the application's logging backend.
+type App struct {
+	*fx.App
+
+	tb TB
```

Comment:
Yep, it's a subset of that interface. Depending on just a subset makes it easier to write a small testing spy, and it also avoids any chance of mistakenly compiling the testing package into production binaries.